lr-cli/cortx_setup/commands/storageset/config/durability.py

```diff
@@ -37,79 +45,104 @@ class DurabilityConfig(Command):
         },
         'type': {
             'type': str,
-            'default': 'sns',
+            'default': None,
             'optional': True,
             'dest': 'durability_type',
+            'choices': ['sns', 'dix'],
             'help': 'Storageset durability type. e.g: {sns|dix}'
         },
         'data': {
             'type': int,
-            'default': 4,
+            'default': None,
             'optional': True,
             'help': 'Storageset durability data'
         },
         'parity': {
             'type': int,
-            'default': 2,
+            'default': None,
             'optional': True,
             'help': 'Storageset durability parity value'
         },
         'spare': {
             'type': int,
-            'default': 2,
+            'default': None,
             'optional': True,
             'help': 'Storageset durability spare'
         }
     }
 
     def run(self, **kwargs):
         try:
-            index = 'storage_durability_index'
+            confstore_config = 'storage_durability_index'
             storage_set_name = kwargs['storage_set_name']
             durability_type = kwargs['durability_type']
+
+            if not (
+                    durability_type and kwargs['data'] and kwargs['parity'] and kwargs['spare']):
+                raise Exception(
+                    "ERROR: Insufficient command parameters type specified")
+
             cluster_id = get_cluster_id()
+            durability_dict = {
+                'data': str(kwargs['data']),
+                'parity': str(kwargs['parity']),
+                'spare': str(kwargs['spare'])
+            }
 
             self.load_conf_store(
-                index,
+                confstore_config,
                 f'json://{CONFSTORE_CLUSTER_FILE}'
             )
 
-            ss_name = Conf.get(index,
-                               f'cluster>{cluster_id}>storage_set[0]>name')
+            storage_set_info = Conf.get(
+                confstore_config, f'cluster>{cluster_id}>storage_set')
+            storage_set_index = -1
+
+            for index, _ in enumerate(storage_set_info):
+                if storage_set_info[index]['name'] == storage_set_name:
+                    storage_set_index = index
+                    break
 
-            if ss_name != storage_set_name:
+            if storage_set_index == -1:
                 raise ValueError(
-                   "Invalid Storageset name provided: "
-                   f"'{storage_set_name}' not found in ConfStore data. "
-                   "First, set with `cortx_setup storageset create` command."
+                    f"Error: Storage-set '{storage_set_name}' not found."
+                    "Action: Configure with cortx_setup storageset create command"
```

Comment:
Change this as follows:
`Action: Configure with 'cortx_setup storageset create' command`

Reply:
Done